tests/bud.bats

```diff
@@ -3509,3 +3509,15 @@ _EOF
   ## exported /run should not be empty
   assert "$count" == "1"
 }
+
+@test "bud-with-mount-like-buildkit" {
+  skip_if_no_runtime
+  skip_if_in_container
+  # Create file in current build context and mount inside RUN step
+  touch input_file
```

Comment:
Why not put this file in ${TESTSDIR}/bud/buildkit-mount and specify that directory as the build context directory?